examples/axios.test.js

```diff
@@ -872,4 +872,13 @@ describe('axios', function() {
       });
     })
   });
+  it('axios-all', async function() {
+    const axiosrequest1 = axios.get(requestone);
```

Comment:
This test will fail because `requestone` isn't defined, I'd just put the URL here. Like `https://httpbin.org/get`